From a71b568bddea7d7875bf1a5a742503c8023f8f95 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Mon, 9 Sep 2013 10:48:20 -0700
Subject: init.rc: import init.composite.rc

This file contains init.rc directives generated at build time by
the mixins mechanism.

Issue: ADEV-221
Category: AOSP improvement
Domain: AOSP-init.rc, mixins
Source: Internal
Upstream-candidate: Yes
Change-Id: Ie1acf68af7babf04e619140d61f7fad7060e1e53
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 rootdir/init.rc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index a0a9813..c13f723 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -8,6 +8,7 @@ import /init.environ.rc
 import /init.usb.rc
 import /init.${ro.hardware}.rc
 import /init.trace.rc
+import /init.composite.rc
 
 on early-init
     # Set init and its forked children's oom_adj.
-- 
1.7.4.1

